repository/src/main/resources/alfresco/repository.properties

```diff
@@ -1,1247 +1,1285 @@
+repo.event2.queue.dequeueThreadPool.priority=1
+repo.event2.queue.dequeueThreadPool.coreSize=1
+repo.event2.queue.dequeueThreadPool.maximumSize=1
+
+
+# MNT-21083
+# --DELETE_NOT_EXISTS - default settings
+system.delete_not_exists.batchsize=100000
+system.delete_not_exists.delete_batchsize=1000
+system.delete_not_exists.read_only=false
+system.delete_not_exists.timeout_seconds=-1
+system.prop_table_cleaner.algorithm=V2
+
+# Configure the system-wide (ACS) settings for direct access urls.
+#
+# Controls whether this feature is available, system wide.
+# For direct access urls to work, the feature needs to be enabled both system-wide and on the individual content-store.
+system.content.directAccessUrl.enabled=false
+# Sets the default expiry time for the direct access url across all Content Stores.
+# Its value cannot exceed the system-wide max expiry time, it can only be lower (startup failure otherwise).
+# This property is mandatory if direct access urls are enabled system-wide - (startup failure otherwise).
+system.content.directAccessUrl.defaultExpiryTimeInSec=30
+# Sets the upper limit for the direct access urls expiry time, meaning a Content Store will be able to override this
+# value but not exceed it, and the same goes for the clients.
+# A service (Java Interface) client will be able to request a direct access url for a custom expiry time but that time
+# can’t exceed this value.
+# If the requested time exceeds the max value, the expiry time reverts to the default configured one.
+# This property is mandatory if direct access urls are enabled system-wide - (startup failure otherwise).
+system.content.directAccessUrl.maxExpiryTimeInSec=300
+
+# Configure the common S3 storage connector content store settings for direct access urls.
+#
+# Controls whether DAUs are enabled on this specific content store.
+# For direct access urls to work, the feature needs to be enabled both system-wide and on the individual content-store.
+connector.s3.directAccessUrl.enabled=false
+# Sets the expiry time for the direct access url in this store, by overriding the global config.
+# If this value exceeds the content store upper limit or the global limit it will fallback to the global configuration.
+# Its value cannot exceed the system-wide max expiry time, it can only be lower (startup failure otherwise).
+# If not set, the default system-wide setting will be used.
+connector.s3.directAccessUrl.defaultExpiryTimeInSec=30
+# The maximum expiry time interval that can be requested by clients - content-store specific setting.
+# Its value cannot exceed the system-wide configuration, it can only be lower (startup failure otherwise).
+# If not set, the default system-wide setting will be used.
+connector.s3.directAccessUrl.maxExpiryTimeInSec=300
+
+# Configure the REST API configuration settings for direct access urls.
+#
+# Controls whether direct access url requests via the REST API are enabled.
+alfresco.restApi.directAccessUrl.enabled=true
```

Comment:
Is there a reason/need for the use of "system." and "alfresco." prefixes?
The ones starting with and just "connector." don't have them and are fine.

Reply:
removed the alfresco prefix